upsert.gemspec

```diff
@@ -46,10 +46,9 @@ Gem::Specification.new do |gem|
     gem.add_development_dependency 'activerecord-jdbcmysql-adapter'
     gem.add_development_dependency 'activerecord-jdbcpostgresql-adapter'
   else
-    gem.add_development_dependency 'activerecord-mysql2-adapter'
     gem.add_development_dependency 'activerecord-postgresql-adapter'
     gem.add_development_dependency 'sqlite3'
-    gem.add_development_dependency 'mysql2'
+    gem.add_development_dependency 'mysql2', '~> 0.3.10'
```

Comment:
Travis was barfing when it wasn't locked (but due to a "already activated mysql2 v3.x error", not due to any actual problem).  If you're able to get it working that'd be awesome :+1:.